spec/javascripts/actions/story_spec.js

```diff
@@ -196,4 +206,17 @@ describe('Story Actions', () => {
       expect(fakeDispatch).toHaveBeenCalledWith(Story.storyFailure(storyId, error));
     });
   });
+
+  describe('highlight', () => {
+    it('Should dispatch updateHighlight', () => {
```

Comment:
This method `highlight` always will dispatch _ updateHighlight_?
If so, you could update this description to `always dispatch updateHighlight`.

Reply:
Yes, always will do that. I will update the description.